src/common/adapters/jira.js

```diff
@@ -1,5 +1,12 @@
 import { $find, $has, $text, $attr } from './helpers';
 
+const TYPES = ['bug', 'chore'];
+const normalizeType = (type) => {
+  const sanitizedType = type && type.toLowerCase();
+  if (TYPES.indexOf(sanitizedType) > -1) return sanitizedType;
```

Comment:
`TYPES.includes(…)` might be nicer but with regard to browser support, your solution is likely better 😁